lib/string.js

```diff
@@ -513,10 +513,7 @@ helpers.replaceFirst = function(str, a, b) {
  * @example {{reverse 'abcde'}} -> edcba 
  */
 
-helpers.reverse = function(str) {
-  if (typeof(str) !== 'string') return '';
-  return str.split('').reverse().join('');
-};
+helpers.reverse = require('./array').reverse;
```

Comment:
We have the reverse helper in array, and one was overriding the other. Getting sure they are the same, handling the types properly